800.renames-and-merges/m.yaml

```diff
@@ -186,6 +186,8 @@
 - { setname: mixxx,                    name: mixxx-qt5, addflavor: true }
 - { setname: mkcert,                   name: "go:mkcert", addflavor: true }
 - { setname: mksh,                     name: mksh-static, addflavor: true }
+- { setname: mldonkey,                 name: [mldonkey-core,mldonkey-gui] }
```

Comment:
`addflavor: true` is needed

Reply:
For future reference why I need to add addflavor: true for this case?